src/order/filter/basic.lean

```diff
@@ -554,6 +554,16 @@ lemma eventually.mono {p q : α → Prop} {f : filter α} (hp : ∀ᶠ x in f, p
   ∀ᶠ x in f, q x :=
 hp.mp (f.eventually_of_forall hq)
 
+lemma eventually.congr {f : filter α} {β : Type*} {u v : α → β} {p : β → Prop}
+  (h : ∀ᶠ a in f, u a = v a) (h' : ∀ᶠ a in f, p (u a)) :
+  ∀ᶠ a in f, p (v a) :=
+eventually.mp h' (eventually.mp h (eventually_of_forall f (by simp {contextual := tt})))
```

Comment:
I'd use `eventually.mono` instead of `eventually.mp h (eventually_of_forall f _))`, same below.

Reply:
Done here. I have not understood what you mean by "same below".